src/main/java/org/sagebionetworks/bridge/spring/controllers/StudyController.java

```diff
@@ -114,8 +114,13 @@ public VersionHolder updateStudyForDeveloper() throws Exception {
 
     @PostMapping("/v3/studies/{identifier}")
     public VersionHolder updateStudy(@PathVariable String identifier) throws Exception {
-        getAuthenticatedSession(ADMIN);
-
+        UserSession session = getAuthenticatedSession(ADMIN);
+        
+        // ADMIN must have permissions to operate on this study.
+        if (!session.getStudyIdentifier().getIdentifier().equals(identifier)) {
+            throw new UnauthorizedException("Study admin cannot update another study");
+        }
```

Comment:
It looks like there are no positive cases for any of these API calls. Should we add those to StudyControllerTest?